website/templates/index.html

```diff
@@ -0,0 +1,122 @@
+{% extends "base.html" %} {% block title %}Add Product{% endblock %} {% block
+content %}
+<div class="container-md vh-100">
+  <h2 class="mt-4 mb-4">Add Product To Inventory</h2>
+  <form
+    id="addprod"
+    action="{{ url_for('add_item') }}"
+    method="post"
+    class="mx-auto"
+  >
+    <div class="form-group row mb-3">
+      <label for="name" class="col-md-6 col-form-label"
+        ><strong>Product Name:</strong></label
+      >
+      <div class="col-md-6">
+        <input
+          type="text"
+          id="name"
+          name="name"
+          value="{{product.name}}"
```

Comment:
Value should be given the user. You should have product.name in the text box. Please fix this